.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+sudo: false
+
+node_js:
+  - '5'
+  - '4'
+
+cache:
+  directories:
+  - node_modules
+
+after_script:
+  npm run coverage
```

Comment:
does _coverage_ script exists in package.json? if not you either add it there or remove it from here